src/Core/composer.json

```diff
@@ -23,6 +23,7 @@
         "psr-4": {
             "AsyncAws\\Core\\": ""
         },
+        "files": ["Test/PHPUnitTestCase.php"],
```

Comment:
I think it's only needed in `Generator` (+ `Aws` because of subtreesplit)  when using Reflection to change the source code of a Test.
When running test, there is no issue, the Class `PHPUnit\TestCase` is well loaded.

People using `Core` + our testCase should not be impacted

Reply:
I've updated the solution now. I pretty much only moved DummyTest to a new file.

Reply:
do not missunderstand my comment. I liked using composer to load the alias. 

I was just worried about loading it for everybody with "production" settings.

Reply:
Yeah. It was. 

I changed that so it is only loaded when the class is used.